pygments/lexers/rell.py

```diff
@@ -0,0 +1,62 @@
+"""
+    pygments.lexers.rell
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for the Rell language.
+
+    :copyright: Copyright 2006-2025 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+""" 
+
+import re
+from pygments.lexer import RegexLexer, bygroups, default
+from pygments.token import Comment, Keyword, Name, String, Number, \
+        Punctuation, Whitespace
+
+__all__ = ['RellLexer']
+
+class RellLexer(RegexLexer):
+    """
+    A Lexer for Rell.
+    """
+    name = 'Rell'
+    url = 'https://docs.chromia.com/rell/rell-intro'
+    aliases = ['rell']
+    filenames = ['*.rell']
+    mimetypes = ['text/x-rell']
+    version_added = '2.19.2'
+
+    ident = r'[a-zA-Z_][a-zA-Z0-9_]*'
+
+    tokens = {
+        'root': [
+            (r'(big_integer|boolean|byte_array|decimal|gtv|integer|json|list|'
+             r'map|mutable|set|text|virtual)\b',
+             Keyword.Type),
+            (r'(false|true|null)\b', Keyword.Constant),
+            (r'(entity|enum|namespace|object|struct)\b', Keyword.Declaration),
+            (r'(function|operation|query)\b', Keyword.Declaration, 'function'),
+            (r'(abstract|and|break|continue|create|delete|else|for|if|import|'
```

Comment:
Please use `words()` instead of providing the words concatenated manually using `|`.